event-log-core/src/main/scala/com/ubirch/models/EventLogRow.scala

```diff
@@ -1,16 +1,16 @@
 package com.ubirch.models
 
-import java.util.Date
-
 import org.json4s.JValue
 
+import java.util.Date
+
 /**
   * Concrete type for the EventLogBase whose type T is JValue
   *
   * @param id            String that identifies the EventLog. It can be a hash or a UUID or anything unique
   * @param customerId    Represents an id for a customer id.
   * @param serviceClass  Represents the name from where the log comes.
-  *                     E.G: The name of the class.
+  *                      E.G: The name of the class.
   * @param category      Represents the category for the event. This is useful for
   *                      adding layers of description to the event.
   * @param event         Represents the event that is to be recorded.
```

Comment:
the explanation of the status field is missing

Reply:
good point! will add it. thanks!